### ISSUE TITLE
WIDER Face accuracy

### ISSUE THREAD

**Issue by NarcissusInMirror** [BeloborodovDS/MobilenetSSDFace#8]
Hi there, thank you for your great job!
I want to know that do you have the test result of all the models on WIDER Face?
I use caffe to test the model you provided on the WIDER Face validation set but get a very low mAP value smaller than 0.01, even though I finetuned it about 45000 iterations, I only got the mAP value around 0.30, and I think there may be something wrong, so can you tell me the best mAP result you got? Thank you very much!

**Comment by BeloborodovDS** [comment]
I don't think I have those results saved anywhere (I do not support this project anymore). However I believe that mAP was at least 0.7 on a mixture of WIDER and FDDB, probably higher. 

Try visualizing predicted boxes vs ground truth boxes - maybe it's just a scaling issue. Whatever the problem might be, visualizing your predictions is always a good idea.

**Comment by NarcissusInMirror** [comment]
@BeloborodovDS Thank you very much for your suggestion! I 'll have a try!
